src/app/components/AdvertisingArea/Banner.scss

```diff
@@ -11,7 +11,7 @@
   color: $color-dark;
```

Comment:
remove unnecessary styles. all styles for banner we should get from API response as is implemented `src/server/dummy-data/baner.json`
@YarynaD could you look at this